lib/app/modules/smart/smart_discovery_sequence.rb

```diff
@@ -108,25 +156,28 @@ class SMARTDiscoverySequence < SequenceBase
           security_info = @conformance.rest.first.security.extension.find { |x| x.url == 'http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris' }
           registration_url = security_info.extension.find { |x| x.url == 'register' }
           registration_url = registration_url.value if registration_url
-          assert !registration_url.blank?, 'No dynamic registration endpoint in conformance.'
+          assert registration_url.present?, 'No dynamic registration endpoint in conformance.'
           assert_valid_http_uri registration_url, "Invalid registration url: '#{registration_url}'"
 
           manage_url = security_info.extension.find { |x| x.url == 'manage' }
           manage_url = manage_url.value if manage_url
-          assert !manage_url.blank?, 'No user-facing authorization management workflow entry point for this FHIR server.'
+          assert manage_url.present?, 'No user-facing authorization management workflow entry point for this FHIR server.'
         end
 
-        @instance.update(oauth_authorize_endpoint: @conformance_authorize_url, oauth_token_endpoint: @conformance_token_url, oauth_register_endpoint: registration_url)
+        @instance.update(
+          oauth_authorize_endpoint: @conformance_authorize_url,
+          oauth_token_endpoint: @conformance_token_url,
+          oauth_register_endpoint: registration_url
+        )
       end
 
       test 'OAuth Endpoints must be the same in the conformance statement and well known endpoint' do
         metadata do
-          id '03'
-          link 'http://www.hl7.org/fhir/smart-app-launch/capability-statement/'
+          id '05'
+          link 'http://hl7.org/fhir/smart-app-launch/conformance/index.html#using-cs'
           desc %(
-
-           If a server requires SMART on FHIR authorization for access, its metadata must support automated discovery of OAuth2 endpoints
-
+           If a server requires SMART on FHIR authorization for access, its
+           metadata must support automated discovery of OAuth2 endpoints
           )
         end
 
```

Comment:
I know you didn't touch this part, but is there a reason we don't also check that the registration endpoint is consistent between the well-known endpoint response and the conformance statement? Seems like we should check that too.

Reply:
I imagine that it's because this test is only testing the required endpoints.

Reply:
That makes sense, but I think it would be good to highlight it to folks so that they are aware in case those URLs are different even if we don't use them directly ourselves.  Doesn't have to be in this PR though.